final_prep/annotations.rb

```diff
@@ -4,19 +4,19 @@
 
 # Build a Bear
 
-def build_a_bear(name, age, fur, clothes, special_power)
-  greeting = "Hey partner! My name is #{name} - will you be my friend?!"
-  demographics = [name, age]
-  power_saying = "Did you know that I can #{special_power}?"
+def build_a_bear(name, age, fur, clothes, special_power) #initializes a method that takes 5 inputs.
```

Comment:
A better way to say this would be "_declares_ a method that takes 5 _parameters_."